src/components/Gnb.tsx

```diff
@@ -31,6 +40,7 @@ export const Gnb = () => {
           {/* <MenuBtn onClick={onClickMenu('/class/' + cid)}>Question List</MenuBtn> */}
           <MenuBtn onClick={onClickSwitchClass}>Classes</MenuBtn>
           <MenuBtn onClick={onClickMyPage}>My Page</MenuBtn>
+          {isAdmin && <MenuBtn onClick={onClickAdmin}>Admin</MenuBtn>}
```

Comment:
I would not make the link to admin page visible (contained in client side code) on the client side for better security.

Reply:
> I would not make the link to admin page visible (contained in client side code) on the client side for better security.

I'm not really sure on how to tackle this problem 😅; what could be a better way to do it?

Reply:
> I would not make the link to admin page visible (contained in client side code) on the client side for better security.

is it something like this https://github.com/kixlab/kuiz/pull/46/commits/57a475225e818d6dc78160da1be139ae578d4f87?

Reply:
oh I meant removing `<MenuBtn onClick={onClickAdmin}>Admin</MenuBtn>`. I think we can work on this in another PR.